test/run

```diff
@@ -16,7 +16,7 @@ source $(dirname $0)/common
 
 cd ${ROOT}
 
-extra_args="--ursula-user=${LOGIN_USER} --ursula-debug --provisioner=heat --heat-stack-name=${testenv_heat_stack_name}"
+extra_args="--ursula-user=${LOGIN_USER} --ursula-debug --ursula-sudo"
```

Comment:
So here we removing --provisioner=heat bcs for running test we really do not need to make heat calls as it needs a stack there with cloud deployed?